contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,103 @@
+
+With lesser devices to target, iOS developers have more control over their application and are able to customise the application to a greater extent, given that there are less variations (in terms of device compatibility) to worry about. OS versioning in iOS is also more structured, with better backwards-compatability across their devices. Users of iOS devices are always immediately prompted whenever a new version of iOS is available for download, regardless of how old your device is. This means that as developers, we have to worry less about compatability issues, and focus more on the application itself.
+
+#### Lucrative market
+
+While Android has higher market share of devices, iOS applications generate much more revenue than Android, [nearly double](https://techcrunch.com/2018/07/16/apples-app-store-revenue-nearly-double-that-of-google-play-in-first-half-of-2018/). This is largely due to the fact that users of iOS devices are generally more likely to spend on apps, and also because the cost for iOS developers to submit their applicaation to the App Store is higher. 
+
+As an independent iOS developer, this means that your iOS application could bring in more revenue for you as compared to an Android one, assuming you choose to monetize it. 
+
+More app revenue for iOS applications also means companies will look to tackle the iOS platform more so than Android, thus the demand for iOS developers will be high. This in turn results in [higher salary for iOS developers compared to their Android counterparts](https://www.fiercewireless.com/developer/ios-developers-earn-roughly-10k-more-than-android-counterparts-study-shows).
+
+#### Higher quality apps
+
+Publishing an app to the Apple App Store is by no means a simple task. First and foremost, there is an annual fee that you have to pay as a developer in order to [have your app published on the App Store](https://developer.apple.com/ios/submit/). Upon submission, the App Store will then test your app for every possible bug and validation errors before they will proceed to publish on the store. This is drastically different from the Android Play Store where approval process is not as stringent. 
+
+This also means that apps on the App Store are of higher quality in general. As iOS developers, we will be required to pay more attention to our application development process to make sure that everything is working and tested well before attempting to publish it up to the App Store. 
```

Comment:
How long is the review process? As a developer can I just **not check** my app for bugs and rely on the app store to check for bugs for me? (Meaning I can just keep resubmitting after fixing one bug every time the app store tells me there is a bug)